public/locales/en.json

```diff
@@ -486,7 +486,7 @@
     "edit": "edit",
     "noProposalsFound": "No proposals found.",
     "proposalCommunityMembers": "Proposals submitted by community members will appear here.",
-    "minimumThreshold": "A minimum threshold of 1,000,000 PNG is required to submit proposals",
+    "minimumThreshold": "A minimum threshold of 1,000,000 {{ pngSymbol }} is required to submit proposals",
```

Comment:
check below screenshot. vote page is not showing PNG symbol now.

![Screenshot 2022-06-13 at 2 16 46 PM](https://user-images.githubusercontent.com/6604146/173315758-e8980fc9-d025-4db6-9b9d-736feec73244.png)